server/src/main/java/org/elasticsearch/inference/InputType.java

```diff
@@ -15,9 +15,8 @@
  */
 public enum InputType {
     INGEST,
-    SEARCH;
-
-    public static String NAME = "input_type";
+    SEARCH,
+    UNSPECIFIED;
```

Comment:
I'm open to other names for this. Maybe `UNKNOWN`?

Reply:
`UNSPECIFIED` is what it is, it's a good name